### PR TITLE
Version 2.13

### PR DESCRIPTION
**Pull request by larskuhtz**
TODO:

* [x] Double check CHANGELOG
* [x] Update next service date to 8 weeks out.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,30 +1,58 @@
...
-## 2.12.2 (2022-02-04)
+## 2.13 (2022-02-18)
 
-This is a new feature and bug fix release for the `Chainweb Rosetta API 1.0.0` ([#1135](https://github.com/kadena-io/chainweb-node/pull/1135)).
+This version replaces all previous versions. Any prior version will stop working
+on **2022-02-24T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
 
-To upgrade, pull the latest docker image or download the binary and restart the node.
+This version will stop working on **2022-04-07T00:00:00Z**.
```
Comment (larskuhtz) [inline CHANGELOG.md:9]: This date is tentative. Need confirmation or update.

File: node/ChainwebNode.hs [hunk 2]
```diff
@@ -474,10 +474,10 @@ pkgInfoScopes =
 -- -------------------------------------------------------------------------- --
 -- main
 
--- SERVICE DATE for version 2.12
+-- SERVICE DATE for version 2.13
 --
 serviceDate :: Maybe String
-serviceDate = Just "2022-02-24T00:00:00Z"
+serviceDate = Just "2022-04-07T00:00:00Z"
```
Comment (larskuhtz) [inline node/ChainwebNode.hs:480]: Tentative new service date (6 weeks). Needs confirmation or update.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,30 +1,58 @@
...
+Changes:
 
- **[Rosetta] Backwards Incompatibility**
-  * Adds related transaction (continuations) information to the /block endpoints.
-  * Adds related operation information to /block endpoints. Gas operations are now linked together, while coin-table operations are linked to each other in the order they appear.
-  * Improved parsing of remediation logs for blocks containing coin v2 and 20 chain forking transactions. Previously, the logs in the coin v2 block were all grouped together into a single request key, but now it will show the different request keys of the remediation transactions.
-  * Re-enabled metadatas. Rosetta testing tooling used to error out when submitting unstructured JSON. This bug has seen been fixed.
+*   [Pact] Enforce miner keyset formats. (#1314)
+*   [Pact] Fix Pact parser to always consume all input. (#1358)
+*   [Pact] More accurate gas cost calculation for modules uploads. (TODO)
```
Comment (larskuhtz) [inline CHANGELOG.md:15]:
```suggestion
*   [Pact] More accurate gas cost calculation for modules uploads. (#1361)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,43 +1,75 @@
...
-## 2.12.3 (2022-02-13)
+## 2.13 (2022-02-18)
 
-This is a new feature release for the `Chainweb Rosetta API 2.0.0` ([#1145](https://github.com/kadena-io/chainweb-node/pull/1145)).
+This version replaces all previous versions. Any prior version will stop working
+on **2022-02-24T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
 
-To upgrade, pull the latest docker image or download the binary and restart the node.
+This version will stop working on **2022-04-07T00:00:00Z**.
```
Comment (larskuhtz) [inline CHANGELOG.md:9]:
```suggestion
This version will stop working on **2022-04-21T00:00:00Z**.
```
```suggestion
This version will stop working on **2022-04-07T00:00:00Z**.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,43 +1,75 @@
...
+To upgrade, pull the latest docker image or download the binary and restart the node.
 
- **[Rosetta] Backwards Incompatibility**
-  * An Operation's metadata no longer returns the Operation's "tx-id" or its account's "total-balance". The Construction API declares intended Operations and looks for these operations in blocks. There is no way for the Construction API to know the "tx-id" and "total-balance" when creating the intended operations.
+Changes:
 
-## 2.12.2 (2022-02-04)
+*   [Pact] Enforce miner keyset formats. (#1314)
+*   [Pact] Fix Pact parser to always consume all input. (#1358)
+*   [Pact] More accurate gas cost calculation for modules uploads. (#1361)
```
Comment (sirlensalot) [inline CHANGELOG.md:17]:
```suggestion
*   [Pact] More accurate gas cost calculation for module uploads. (#1361)
```

**Review by LindaOrtega — APPROVED** [review]
Looks good! Approved with non-blocking edit suggestions

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,43 +1,75 @@
...
+To upgrade, pull the latest docker image or download the binary and restart the node.
 
- **[Rosetta] Backwards Incompatibility**
-  * An Operation's metadata no longer returns the Operation's "tx-id" or its account's "total-balance". The Construction API declares intended Operations and looks for these operations in blocks. There is no way for the Construction API to know the "tx-id" and "total-balance" when creating the intended operations.
+Changes:
 
-## 2.12.2 (2022-02-04)
+*   [Pact] Enforce miner keyset formats. (#1314)
+*   [Pact] Fix Pact parser to always consume all input. (#1358)
+*   [Pact] More accurate gas cost calculation for module uploads. (#1361)
```
Comment (sirlensalot) [inline CHANGELOG.md:17]:
```suggestion
*   [Pact] Improve gas cost calculations. (#1361,#1369)
```

**Review by sirlensalot — APPROVED** [review]

**Pull request merged**